src/Domain.LinnApps/PurchaseOrderDetails.cs

```diff
@@ -12,10 +14,10 @@ public class PurchaseOrderDetail
 
         public string PartNumber { get; set; }
 
-        public PurchaseOrder PurchaseOrder { get; set; }
-
         public PurchaseOrderDelivery PurchaseDelivery { get; set; }
```

Comment:
there can be multiple deliveries per order line. I don't know if this matters for the report but would be wrong if this becomes the actual model. (if it is then it should be in the purchase orders folder)

Reply:
Ah yeah, good point. I think I'll need to rewrite a few lines slightly and work out if it's a delivery per row of report or if a row will use sum of delivery qtys etc. Think possibly a row per delivery because it takes dates from the delivery. I'll note this + to use the nice testing methods into an issue for myself when I'm back